RELEASE_NOTES.md

```diff
@@ -8,6 +8,8 @@
 - Increase test coverage (#524)
 - Add sql query to retrieve only last values of entities (#500)
 - Support configuration of back off factor for CrateDB (#503)
+- Added exception handling and updated response where 
```

Comment:
there is a space after `where` that needs to be remove for the linter test to pass